dev-tools/Dockerfile

```diff
@@ -20,4 +20,4 @@ ARG PYTHON_BUILD_DEPS=libffi-dev,python3-dev,py3-virtualenv
 RUN apk add $(echo -n "$PYTHON_BUILD_DEPS" | tr , ' ')
 
 # Install test tools (Ragger framework, Speculos emulator, Ledgerblue...)
-RUN pip3 install ragger[tests,all_backends]==1.16.0
+RUN pip3 install --no-cache-dir "ragger[tests,all_backends]~=1.16"
```

Comment:
I think I prefer an explicit version so that we immediately now if a patch is included ina  specific version but as you want.

Reply:
I understand with the goal, but at the same time I think it is too much to have to explicit the current version every time for patches. I think the version can be retrieved "simply enough" with the image.

But following this reasoning, I guess I could question this need of specifying a specific version. What was the bug we encountered that led us to pin it?

Reply:
(Short-term I'll resolve and deploy but the question remains)